docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,865 @@
+Process Costing Library
```

Comment:
From my first impressions, this file appears to be a good summary of the example costing package based on SSLW, but we need to have a general introduction to the costing framework before this. It is critical for costing that we emphasize that this is a general framework and that the SSLW example is just an example to guide users in developing their own packages. Every time we have broached costing with industry before the response has been "we have our own methods so we are not interested" - we need to make it clear that this framework allows them to integrate their costing methods with IDAES models.

I think there need to be 3 parts to this:

The first part should cover the general framework, what it does and how to use it. For this part, there should be no mention of specific costing packages, but instead say that users need to provide a costing package (with a link to the second part). Some generic (non-functional) examples could appear here.

The second part would be a library of available costing packages - for now it will be relatively sparse (the SSLW package and maybe something used by the power plant team). This page would be mainly links to separate pages for each package.

For each costing package, there would then be a page discussing the details, along with the actual correlations and  parameters used.